package exception;

public class TestThrow1 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.err.println("begin");
		throw new RuntimeException();
		System.out.println("end");
	}

}
